### ISSUE TITLE
Drop-down menu doesn't appear

### ISSUE THREAD

**Issue by khughitt** [darrenburns/textual-autocomplete#1]
Greetings!

I'm trying to run the examples for the latest version of textual/textual-autocomplete, but have been unable to get the menus to render as expected.

When attempting to run either of the demos, I get a usable input field and tab completion appears to work, but the autocomplete options are never shown:

https://user-images.githubusercontent.com/125001/209702957-b19bc620-ed7d-4749-a689-0d70c7119d32.mp4

This is on Arch / kitty / zsh. I tried a few other terminals, just in case (Alacritty, xterm, urxvt), but no luck. Same result in Bash/ZSH.

I also tried installing textual in a clean conda environment and running the examples from there with the same result.

I tried running the tests for textual, and there are a few errors there, but none that are obviously related to the autocomplete issue:

```
========================================================= short test summary info ==========================================================
ERROR tests/devtools/test_devtools.py::test_devtools_valid_client_log
ERROR tests/devtools/test_devtools.py::test_devtools_string_not_json_message
ERROR tests/devtools/test_devtools.py::test_devtools_invalid_json_message
ERROR tests/devtools/test_devtools.py::test_devtools_spillover_message
ERROR tests/devtools/test_devtools.py::test_devtools_console_size_change
ERROR tests/devtools/test_devtools_client.py::test_devtools_client_is_connected
ERROR tests/devtools/test_devtools_client.py::test_devtools_log_places_encodes_and_queues_message
ERROR tests/devtools/test_devtools_client.py::test_devtools_log_places_encodes_and_queues_many_logs_as_string
ERROR tests/devtools/test_devtools_client.py::test_devtools_log_spillover
ERROR tests/devtools/test_devtools_client.py::test_devtools_client_update_console_dimensions
ERROR tests/devtools/test_redirect_output.py::test_print_redirect_to_devtools_only
ERROR tests/devtools/test_redirect_output.py::test_print_redirect_to_logfile_only
ERROR tests/devtools/test_redirect_output.py::test_print_redirect_to_devtools_and_logfile
ERROR tests/devtools/test_redirect_output.py::test_print_without_flush_not_sent_to_devtools
ERROR tests/devtools/test_redirect_output.py::test_print_forced_flush_sent_to_devtools
ERROR tests/devtools/test_redirect_output.py::test_print_multiple_args_batched_as_one_log
ERROR tests/devtools/test_redirect_output.py::test_print_strings_containing_newline_flushed
ERROR tests/devtools/test_redirect_output.py::test_flush_flushes_buffered_logs
===================================== 699 passed, 1 skipped, 4 xfailed, 1 warning, 18 errors in 18.13s =======
```

I had a friend check on another machine (also Arch), and the demos worked fine for him, so it's something about my setup that autocomplete is not liking.

Any ideas what the issue might be or how I can get more information to localize the problem?

I'm excited to see your work on this though! A solid autocomplete is something TUI frameworks have been sorely missing.

Cheers,
Keith

**System info**

- Arch Linux
- Python 3.10.9
- textual 0.8.1
- textual-autocomplete 3a20f092b53e0d272b0c1e573768a5a0b86e7e4d
- zsh 5.9
- kitty 0.26.5

**Comment by jgoodson** [comment]
A quick check indicates the examples work with Textual 0.5.0, but the above behavior happens with 0.6.0 or newer. On newer Textual versions, the Dropdown elements appear to be created with no width:

```
[09:20:17] EVENT                                                                                                                                                                                                              message_pump.py:454
Resize(size=Size(width=0, height=15), virtual_size=Size(width=0, height=15)) >>> DropdownChild() method=None
[09:20:17] EVENT                                                                                                                                                                                                              message_pump.py:454
Show() >>> DropdownChild() method=None
```

**Comment by byt3bl33d3r** [comment]
@darrenburns do you have any starting points as to how to fix this ? Can make a PR if necessary.

**Comment by darrenburns** [comment]
This should be fixed as of 1.0.1b0, which is now available on PyPI. Let me know if you encounter any issues.
